webapp/store/selectors/nav.js

```diff
@@ -129,15 +128,10 @@ export const sortedSidebarItems = createSelector(
 );
 
 // Useful selector for the Panel which just needs to match names to paths
-export const uniquePathsByName = createSelector(
-  getMetadataList,
-  metadata =>
-    metadata
-      .filter(plugin => plugin.pathName)
-      .reduce(
-        (paths, { name, pathName }) => ({ ...paths, [name]: pathName }),
-        {}
-      )
+export const uniquePathsByName = createSelector(getMetadataList, metadata =>
+  metadata
+    .filter(plugin => plugin.pathName)
```

Comment:
Reduce can be any combination of map and filter but its nbd here since its small data